lib/internal/crypto/random.js

```diff
@@ -281,10 +282,32 @@ function getRandomValues(data) {
   return data;
 }
 
+const hexDigits = [
+  48, 49, 50, 51, 52, 53, 54, 55,
+  56, 57, 97, 98, 99, 100, 101, 102
+];
+
+function randomUUID() {
+  const data = getRandomValues(new Uint8Array(16));
+  data[6] = (data[6] & 0x3f) | 0x80;
+  data[8] = (data[8] & 0x0f) | 0x40;
+
+  const ret = Buffer.alloc(36);
```

Comment:
Couldn't this be lazily created once and reused across multiple requests?

Same goes with the `Uint8Array` at the top of this function.

Reply:
Certainly could yes.